src/components/calcite-slider/calcite-slider.scss

```diff
@@ -7,23 +7,25 @@ $calcite-slider-track-height: 2px;
 $calcite-slider-tick-height: 4px;
 
 :host {
-  --calcite-slider-spot: #{$ui-blue};
   --calcite-slider-tick: #{$blk-100};
-  --calcite-slider-label: #{$blk-160};
-  --calcite-slider-track: #{$blk-050};
+  --calcite-slider-label: #{$ui-text-3};
+  --calcite-slider-track: #{$blk-040};
   --calcite-slider-handle: #{$blk-000};
+  --calcite-slider-handle-border: #{$blk-140};
+  --calcite-slider-handle-precise: #{$blk-140};
```

Comment:
Seems like this is a place where just using `--calcite-slider-handle-border` would work, it's more general than having a variable just for a single modifier and the values are the same.

Reply:
updated